pkg/output/command.go

```diff
@@ -54,6 +55,10 @@ func PrintError(err error, l *zap.SugaredLogger) {
 	if errors.Is(err, common.ErrExitWithError) {
 		return
 	}
+	if errors.Is(err, terminal.InterruptErr) {
+		l.Info("The command execution is interrupted")
+		return
+	}
```

Comment:
Graceful shutdown for interrupting the wizard, otherwise an ugly stack trace is printed.